addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1038,7 +1038,17 @@ const Wysiwyg = Widget.extend({
 
             $odooFields.each((i, field) => {
                 const observer = new MutationObserver((mutations) => {
-                    mutations = this.odooEditor.filterMutationRecords(mutations);
+                    mutations = this.odooEditor.filterMutationRecords(mutations).filter(mutation => {
+                        // Ignore empty bootstrap attributes added to a link
+                        // when its popover initialized.
+                        if (mutation.type === "attributes") {
+                            // TODO: adapt to "data-bs-original-title" on 16.0.
+                            if (["title", "data-original-title"].indexOf(mutation.attributeName) !== -1
+                                && !mutation.oldValue && mutation.target.matches('a[data-original-title=""], a[title=""]')) {
+                                return false;
+                            }
+                        }
+                    });
```

Comment:
Am I missing something or you forgot to `return true;` at the end of the function meaning that your code here is the same as `mutations = [];` and thus the whole synchronization thing does not work anymore? :grimacing:

Reply:
@qsm-odoo Good catch! Actually, this was also misleading my tests on `17.0` :grimacing: (when all mutations were removed in the `filter()`, the fix seemed to work correctly, but in fact, the whole synchronization mechanism was disabled, as you said :man_facepalming:).

So I fixed that and retested in `17.0` and what I was afraid of, happened!: It seems that not only the "bootstrap attributes" mutations are guilty here but also other mutations (coming from code ignored in the main observer by `observerUnactive(...)` [A]) are triggering the issue :grimacing:

@bvr-odoo In this situation (since we can't add filters for all mutations from [A] one by one), I think the cleanest solution left to fix the issue here, is to prevent these temporary rendering classes from being applied on fields synchronization :+1: I updated the code to add the new implementation. Just let me know if something needs to be changed here :+1: